src/test/java/com/epam/HelloUnitTest.java

```diff
@@ -1,10 +1,12 @@
 package com.epam;
 
+import lombok.extern.slf4j.Slf4j;
 import org.junit.jupiter.api.Test;
-
-public class HelloUnitTest {
+@Slf4j
+class HelloUnitTest {
         @Test
-        public void testSomething() {
+        void testSomething() {
+                log.info("Test");
                 System.out.println("HelloUnitTest");
```

Comment:
This `sout` doesn't make any sense. You can delete it

Reply:
Done